src/components/addRemElemTest/AddRemoveElemTest.vue

```diff
@@ -53,22 +53,21 @@ export default class AddRemoveElementTest extends Vue {
 	}
 
 	maxindex(): number {
-		let i = this.todos.todosAmount;
-		return i > 0 ? i - 1 : 0;
+		const todosAmount = this.todos.todosAmount;
```

Comment:
use spaces instead of tabs (4 spaces)
```suggestion
        const todosAmount = this.todos.todosAmount;
```